VideoApp/VideoApp/Video/VideoAppCameraSource.swift

```diff
@@ -85,41 +95,20 @@ import TwilioVideo
     private func startCameraSource() {
         guard let cameraSource = cameraSource else { return }
 
-        let targetSize: CMVideoDimensions
-        let cropRatio: CGFloat
-        let frameRate: UInt
-        
-        switch appSettingsStore.videoCodec {
-        case .h264, .vp8:
-            // 640 x 480 squarish crop (1.13:1)
-            targetSize = CMVideoDimensions(width: 544, height: 480)
-            
-            cropRatio = CGFloat(targetSize.width) / CGFloat(targetSize.height)
-            frameRate = 20
-        case .vp8Simulcast:
-            // 1024 x 768 squarish crop (1.25:1) on most iPhones. 1280 x 720 squarish crop (1.25:1) on the iPhone X
-            // and models that don't have 1024 x 768.
-            targetSize = CMVideoDimensions(width: 900, height: 720)
-            
-            cropRatio = CGFloat(targetSize.width) / CGFloat(targetSize.height)
-            frameRate = 24 // With simulcast enabled there are 3 temporal layers, allowing a frame rate of f/4
-        }
-        
         let captureDevice = CameraSource.captureDevice(position: .front)!
-        let preferredFormat = selectVideoFormatBySize(captureDevice: captureDevice, targetSize: targetSize)
-        preferredFormat.frameRate = min(preferredFormat.frameRate, frameRate)
+        let preferredFormat = selectVideoFormat(captureDevice: captureDevice, isMultiparty: false)
         
         let cropDimensions: CMVideoDimensions
         
         if preferredFormat.dimensions.width > preferredFormat.dimensions.height {
             cropDimensions = CMVideoDimensions(
-                width: Int32(CGFloat(preferredFormat.dimensions.height) * cropRatio),
+                width: Int32(CGFloat(preferredFormat.dimensions.height) * videoCodec.cropRatio),
```

Comment:
How come we only do this crop stuff here and not in other places like when the camera is switched from front to back?